### PR TITLE
Add scripts to rule them all

### PR DESCRIPTION
**Pull request by pezholio**
This adds some simple scripts to aid running in development. We use these in the frontend application, so it’s useful to mirror this approach in the API too, to aid with context switch. Currently the bootsrap and server scripts are stubs, as there’s nothing we (yet) need to do to bootstrap / setup the app, but they’re worth having around as a reminder to update them when we do start introducing external dependencies, like a database etc.

### CODE REVIEW

**Comment by pezholio** [comment]
[Add scripts to rule them all for the API](https://trello.com/c/EHsjukQc/470-add-scripts-to-rule-them-all-for-the-api)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+
+```bash
+script/setup
+```
+
+If you're coming back to the application after a certain amount of time, you can run:
+
+```bash
+script/bootstrap
+```
```
Comment (edavey) [inline README.md:21]: As these don't do anything yet would it be helpful to say as much, e.g. with a "TODO"

File: script/bootstrap [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# script/bootstrap: Resolve all dependencies that the application requires to
+#                   run.
+
+set -e
+
+cd "$(dirname "$0")/.."
```
Comment (edavey) [inline script/bootstrap:8]: should this print out something to acknowledge that this is a placeholder script e.g. "Bootstrap script not yet implemented"

**Review by edavey — APPROVED** [review]
Good stuff: thanks for adding these. 

I've made a couple of comments/suggestions asking whether it's worth noting explicitly that bootstrapping isn't yet implemented...

**Review by richpjames — APPROVED** [review]
Looks good, agree with both of Ed's points

**Pull request merged**